content/docs/role/_index.md

```diff
@@ -2,7 +2,7 @@
 weight: 500
 title: "Role"
 description: "Definition of the role, key responsibilities and types of Software Architects "
-icon: "design_services"
+icon: "person"
```

Comment:
```suggestion
icon: "badge"
```

Looks more suitable for me

Reply:
I do not have a preference. What do you think @gekola ?

Reply:
`"badge"` seems better to me.

Reply:
changed